mcs/class/referencesource/mscorlib/system/math.cs

```diff
@@ -62,7 +62,233 @@ public static class Math {
       [ResourceExposure(ResourceScope.None)]
       [MethodImplAttribute(MethodImplOptions.InternalCall)]
       public static extern double Ceiling(double a);      
-      
+
+#if MONO
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static byte Clamp(byte value, byte min, byte max)
+        {
```

Comment:
This should use K&R indentation.

Reply:
I copy pasted this from CoreCLR, I rather not change the style of the copy-pasted code.